src/main/java/core/basesyntax/ApplicationStarter.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.db.Storage;
+
+public class ApplicationStarter {
+    public static void main(String[] args) {
+        ConsoleHandler consoleHandler = new ConsoleHandler();
+        consoleHandler.handle();
+
+        Storage.bets.forEach(bet ->
+                System.out.println("Value: " + bet.getValue() + ", risk: " + bet.getRisk()));
```

Comment:
If you have multiple db's, let's output them all

Reply:
it would be easier to override toString() for Bet